feature-group-definitions/broadcast-channel.yml

```diff
@@ -1,4 +1,5 @@
 name: BroadcastChannel
+description: The `BroadcastChannel` interface allows sending messages between same-origin browsing contexts, for example between the same page loaded in multiple tabs.
```

Comment:
style nits:

- avoiding "allows"
- maybe this is idiosyncratic but I'm not sure "for example" is formally correct as a coordinating conjunction (brought to you because TIL that the equivalent in Dutch _is_)

```suggestion
description: The `BroadcastChannel` interface sends messages between same-origin browsing contexts, such as between the same page loaded in multiple tabs.
```

Reply:
I like such as. I'll update two existing cases for "for example" as conjunction.

Reply:
https://github.com/web-platform-dx/web-features/pull/742

Reply:
And I'll apply this part of the suggestion:

```suggestion
description: The `BroadcastChannel` interface allows sending messages between same-origin browsing contexts, such as between the same page loaded in multiple tabs.
```